Atomic/atomic.py

```diff
@@ -915,42 +917,26 @@ def _parse_imagename(imagename):
     def _convert_to_skopeo(image):
         i = image.replace("oci:", "")
         if "http:" in i:
-            return ["--insecure", "docker://" + i.replace("http:", "")]
+            return ["--insecure"], "docker://" + i.replace("http:", "")
```

Comment:
yes, true, it ends up here only if users specify "http:" and that should not happen, yet.